src/NuGet.Core/NuGet.ProjectModel/RestoreAuditProperties.cs

```diff
@@ -127,6 +135,7 @@ internal RestoreAuditProperties Clone()
                 EnableAudit = EnableAudit,
                 AuditLevel = AuditLevel,
                 AuditMode = AuditMode,
+                SuppressedAdvisories = SuppressedAdvisories,
```

Comment:
We changed to using HashSets instead of Lists. We considered using an `IReadOnlySet`, but this is only available on .NET 5+ frameworks, so we'll need to use a HashSet instead. 

I've changed the `Clone()` method so it should work correctly now, but let me know if it looks wrong.